RELEASE_NOTES.md

```diff
@@ -1,6 +1,11 @@
 Search Service MKII release notes
```

Comment:
I have full coverage for the change. I am not sure if it is worth the time since there are existing unit tests that test the base functionality.  If you want this I'll have to spend more time coding up these unit tests. I also feel we should have at least one integration test per type that we can use to test against the three or four environments we have.

Reply:
> I am not sure if it is worth the time since there are existing unit tests that test the base functionality.

I don't think there are unit tests that test the recursion are there?

Reply:
Not specifically the recursion but there are tests in indexer worker that exercise the recursion. guidNotFoundError() for example.